script.ts

```diff
@@ -1,99 +1,108 @@
-import { PrismaClient } from "@prisma/client";
-const prisma = new PrismaClient()
+import { PrismaClient } from '@prisma/client';
+const prisma = new PrismaClient();
 
 async function main() {
-
-  await prisma.job_table.deleteMany() // delete the existing data everytime
-  await prisma.department_table.deleteMany() // delete the existing data everytime
-  await prisma.location_table.deleteMany() // delete the existing data everytime
-  await prisma.hr_employees_table.deleteMany() // delete the existing data everytime
+  // await prisma.job_table.deleteMany() // delete the existing data everytime
```

Comment:
Please remove commented out code.

Reply:
We have a reason for this. We can loop you in on site and fix it later.

Reply:
Gotcha!